.chloggen/remove-kubeletstats-utilization-converter.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I changed it from breaking to bug_fix since users the feature gate was already stable, so there should not be users explicitly enabling the old metrics